lib/wallet/cardano-wallet.cabal

```
@@ -243,6 +243,8 @@ library
     Cardano.Wallet.DB.Store.Checkpoints
     Cardano.Wallet.DB.Store.DeltaUTxO.Model
     Cardano.Wallet.DB.Store.DeltaUTxO.Model.Internal
+    Cardano.Wallet.DB.Store.DeltaUTxO.TxOutCBOR
+    Cardano.Wallet.DB.Store.DeltaUTxO.Store
```

```suggestion
    Cardano.Wallet.DB.Store.DeltaUTxO.Store
    Cardano.Wallet.DB.Store.DeltaUTxO.TxOutCBOR
```

(Alphabetical order. 🤷)